Rocket.Chat/Resources/de.lproj/Localizable.strings

```diff
@@ -194,4 +194,11 @@
 "myaccount.settings.title" = "Einstellungen";
 "myaccount.settings.contactus" = "Kontaktiere uns";
 "myaccount.settings.license" = "Lizenz";
+"myaccount.settings.appicon" = "Application Icon"; // TODO:
```

Comment:
App Icon would be sufficient :)
"Anwendungs Icon" sounds not that well as I think everybody uses "App"

Reply:
👍